apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/step-based-flow.tsx

```diff
@@ -78,7 +78,7 @@ interface AuthenticationFlowPropsInterface extends TestableComponentInterface {
     /**
      * Callback to trigger IDP create wizard.
      */
-    onIDPCreateWizardTrigger: (type: string, cb: () => void) => void;
+    onIDPCreateWizardTrigger: (type: string, cb: () => void, template?: any) => void;
```

Comment:
Let's get rid of `any` in a different PR.

```suggestion
    onIDPCreateWizardTrigger: (type: string, cb: () => void, template?: IdentityProviderTemplateInterface | IdentityProviderTemplateGroupInterface) => void;
```